package.json

```diff
@@ -12,11 +12,12 @@
     "@asyncapi/diff": "^0.4.0",
     "@asyncapi/generator": "^1.9.12",
     "@asyncapi/modelina": "^0.59.8",
-    "@asyncapi/parser": "^1.17.0",
+    "@asyncapi/parser": "^2.0.0-next-major.6",
```

Comment:
and here we have the same problem we have with other PR from Jonas. This will be overwritten with next parser release back to `1.18`